app/models/ability.rb

```diff
@@ -0,0 +1,39 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+    user ||= User.new # guest user (not logged in)
+    if user.role == 'admin'
+      can :manage, :all
```

Comment:
[Optional]
- [ ] You can create `admin?` method in your user model to check user role. This way you can keep your code cleaner. 

```ruby
   def admin?
     role == 'admin' ? true : false
   end
```

```ruby
         if user.admin? 
         can :manage, :all
```

Reply:
@sinansevgi  Thank you for letting us know. I created admin? method on our user model. 💪